src/main/resources/messages/en.yml

```diff
@@ -28,12 +28,24 @@
 #                    example: Notch
 # %channelname%:    name of the channel that the message is coming from
 #                    example: server-chat
+# %reply%:          the message displayed when the message is a reply to another message
+#                    message format can be configured with DiscordToMinecraftMessageReplyFormat
+#                    is not included if the message isn't a reply
 #
 # DiscordToMinecraftAllRolesSeparator: the separator used in between roles in %allroles%
 #
-DiscordToMinecraftChatMessageFormat: "[<aqua>Discord</aqua> | %toprolecolor%%toprolealias%<reset>] %name% » %message%"
-DiscordToMinecraftChatMessageFormatNoRole: "[<aqua>Discord</aqua>] %name% » %message%"
+# DiscordToMinecraftMessageReplyFormat: the format for the message displayed to indicate that the message is a reply
+#
+# Available placeholders:
+# %name%:          effective name on Discord (nickname if present, username otherwise) for the user who is being replied to
+#                   example: NotchIsMe
+# %username%:      username on Discord for the user who is being replied to
+#                   example: Notch
+#
```

Comment:
A placeholder for the replied message is missing

Reply:
Hovering to show things is possible with MiniMessage, https://docs.adventure.kyori.net/minimessage.html#hover

Reply:
TIL. I'll add that to the default value as well

Reply:
I've added the placeholder and made the message display on hover, but since this will only work with non-legacy DiscordToMinecraftChatMessageFormat values I think it might be worth saying as much somewhere so that we don't get the inevitable "help the hover isn't working"

Alternatively we could just leave this out of the default value

Reply:
Those options will not get updated with %reply% automatically, either. Arguably the available formats should be documented way better, but it is what ut is

Reply:
Added a comment to tell users that they must use MiniMessage formatting for the hover text to work, though I think somehow converting these values from legacy to MiniMessage would be a better way of doing it.